internal/gen/connect/ping/v1/pingv1connect/ping.connect.go

```diff
@@ -12,10 +12,15 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// The canonical location for this file is
```

Comment:
Is one benefit now for people not using our license header tools is that the license in the .proto file will be included?

Reply:
It does look like that is the result - here's a run from the plugins repo from two images exported w/ `buf export`: https://gist.github.com/pkwarren/0628b6e2412d7f6483e59c23344d9079

Reply:
Yep! This seemed like a good thing to me - do you think it's ok?